Pinta.Core/Managers/WorkspaceManager.cs

```diff
@@ -259,6 +291,10 @@ public void CloseDocument (
 			}
 		}
 
+		document.Layers.LayerAdded += Document_LayerAdded;
```

Comment:
Should these have been `-=` to remove the events?

Reply:
Yup @cameronwhite , my mistake. I just corrected it